tests/queries/0_stateless/02813_float_parsing.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+    toFloat64('1.7091'),
+    toFloat64('1.5008753E7'),
+    toFloat64('6e-09'),
+    toFloat64('6.000000000000001e-9'),
+    toFloat32('1.7091'),
+    toFloat32('1.5008753E7'),
+    toFloat32('6e-09'),
+    toFloat32('6.000000000000001e-9')
+SETTINGS precise_float_parsing = 1;
```

Comment:
would be nice to have the same select with `precise_float_parsing = 0` to see a difference...

Reply:
done